functions/openhab.sh

```diff
@@ -2,44 +2,53 @@
+  Please be aware that downgrading from a newer testing (milestone) build is not officially supported. Please consult with the documentation or community forum and be sure to take a full openHAB configuration backup first!"
+successtext_testing="The milestone (testing) release of openHAB is now installed on your system. Please test the correct behavior of your setup. You might need to adapt your configuration, if available. If you did changes to files below '/var/lib/openhab2' before, they were replaced but you can restore them from backup files next to the originals.
+Check the \"openHAB Release Notes\" and the official announcements to learn about additons, fixes and changes."
   introtext_unstable="Proceed with caution!\\nYou are about to switch over to the latest openHAB 2 unstable snapshot build. The daily snapshot builds contain the latest features and improvements but might also suffer from bugs or incompatibilities. Please be sure to take a full openHAB configuration backup first!"
-  successtext_unstable="The latest unstable snapshot build of openHAB 2 is now running on your system. Please test the correct behavior of your setup. You might need to adapt your configuration, if available. If you did changes to files below '/var/lib/openhab2' before, they were replaced but you can restore them from backup files next to the originals.\\nIf you find any problem or bug, please report it and state the snapshot version you are on. To stay up-to-date with improvements and bug fixes you should upgrade your packages (the openhab2 package) regularly."
+  successtext_unstable="The latest unstable snapshot build of openHAB 2 is now running on your system. Please test the correct behavior of your setup. You might need to adapt your configuration, if available. If you did changes to files below '/var/lib/openhab2' before, they were replaced but you can restore them from backup files next to the originals.\\nIf you find any problem or bug, please report it and state the snapshot version you are on. To stay up-to-date with improvements and bug fixes you should upgrade your packages (the openhab2 and openhab2-addons packages) regularly."
 
   if [ "$1" == "unstable" ]; then
     UNSTABLE=1
   fi
 
+  if [ "$1" == "testing" ]; then
+    TESTING=1
+  fi
+
   if [ ! -n "$UNSTABLE" ]; then
-    echo -n "$(timestamp) [openHABian] Installing or upgrading to latest openHAB release (stable)... "
-    introtext=$introtext_stable
-    successtext=$successtext_stable
+    if [ ! -n "$TESTING" ]; then
+      echo -n "$(timestamp) [openHABian] Installing or upgrading to latest openHAB release (stable)... "
+      introtext=$introtext_stable
+      successtext=$successtext_stable
+      REPO="deb https://dl.bintray.com/openhab/apt-repo2 stable main"
+    else
+      echo -n "$(timestamp) [openHABian] Installing or upgrading to latest openHAB release (milestone)... "
```

Comment:
What about:
`[openHABian] Installing or upgrading to latest openHAB milestone release (testing)` ?